Content.Server/Station/StationSystem.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Maps;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+
+namespace Content.Server.Station;
+
+/// <summary>
+/// System that manages the jobs available on a station, and maybe other things later.
+/// </summary>
+public class StationSystem : EntitySystem
+{
+    [Dependency] private GameTicker _gameTicker = default!;
+    private uint _idCounter = 1;
+
+    private Dictionary<StationId, StationInfoData> _stationInfo = new();
+    /// <summary>
+    /// List of stations for the current round.
+    /// </summary>
+    public IReadOnlyDictionary<StationId, StationInfoData> StationInfo => _stationInfo;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<GameRunLevelChangedEvent>(OnRoundEnd);
+    }
+
+    /// <summary>
+    /// Cleans up station info.
+    /// </summary>
+    private void OnRoundEnd(GameRunLevelChangedEvent eventArgs)
+    {
+        if (eventArgs.New == GameRunLevel.PostRound)
+            _stationInfo = new();
+    }
+
+    public class StationInfoData
+    {
+        public readonly string Name;
+
+        /// <summary>
+        /// Job list associated with the game map.
+        /// </summary>
+        public readonly GameMapPrototype MapPrototype;
+
+        /// <summary>
+        /// The round job list.
+        /// </summary>
+        private readonly Dictionary<string, int> _jobList;
+
+        public IReadOnlyDictionary<string, int> JobList => _jobList;
+
```

Comment:
why not just have this be on the station component tbh

Reply:
it'd also be nice if the stationinfo dict above was just made into data on the component. then entityquery over every stationcomponent (which shouldnt be expensive, theres not very many)

Reply:
definitely do this actually

Reply:
> why not just have this be on the station component tbh

Simple reason: grid splitting.
This makes sure that a station that gets split in two is still considered one station.